src/components/CohortBuilder/Categories/Category.js

```diff
@@ -54,6 +52,21 @@ export default class Category extends React.Component {
     });
   }
 
+  onOntologyClicked = () => {
+    const isModalVisible = true;
+    this.setState({
+      isOntologyModalVisible: isModalVisible,
```

Comment:
Why not using false or true since the property name already explains the context?